.github/workflows/ci.yml

```diff
@@ -25,3 +25,11 @@ jobs:
         run: |
           rustup default 1.43.1
           cargo test
+  build-macos-universal:
+    runs-on: macos-11.0
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install targets
+        run: rustup update && rustup target add x86_64-apple-darwin aarch64-apple-darwin
+      - name: Make App
+        run: make universal
```

Comment:
We don't need to build a universal image here, all we need to do is test the arm code. We're already testing x86, there's no point in doing it twice.

Reply:
By 'test' you just mean to build it, right?

Reply:
Yes, building is sufficient. In fact, all we need is `cargo check --target=aarch64-apple-darwin`.